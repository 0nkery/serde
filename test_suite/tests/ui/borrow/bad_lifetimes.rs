#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
struct Test<'a> {
    #[serde(borrow = "zzz")]
    s: &'a str,
}

fn main() {}
