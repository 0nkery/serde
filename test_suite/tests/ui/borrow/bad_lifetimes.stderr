error: failed to parse borrowed lifetimes: "zzz"
 --> $DIR/bad_lifetimes.rs:6:22
  |
6 |     #[serde(borrow = "zzz")]
  |                      ^^^^^

error: aborting due to previous error

